OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <title type="complete" ref="LIT3600Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                    
+                                    <incipit type="inscriptio" xml:lang="gez">
+                                        <locus target="#53ra"/>
+                                        <hi rend="rubric">ተአምሪሃ፡ ለእግዝእትነ፡ ቅድስት፡ ድንግል፡ በ፪ማርያም፡</hi>
+                                        ወላዲተ፡ አምላክ።
+                                    </incipit>
+                                    
+                                    <incipit type="supplication">
+                                        <locus target="#53ra"/>
+                                        ጸሎታ፡ ወበረከታ፡ ወምሕረተ፡ ፍቁር፡ ወልዳ፡ የሃሉ፡ ምስለ፡ 
+                                        ገብራ፡ 
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ ክርስቶስ፡</hi></persName>
+                                        ለዓለመ፡ ዓለም፡ አሜን<hi rend="rubric">፨</hi>
+                                    </incipit>
+                                    
+                                    <incipit xml:lang="gez">
+                                        <locus from="53ra" to="53rb"/>
+                                        ወሀሎ፡ <hi rend="rubric">፩</hi>ቀሲስ፡ በሀገረ፡ ምስር፡ ዘስሙ፡ ዮሐንስ፡ በከንሲ<hi rend="rubric">፨</hi>
+                                        ወይነብር፡ ውስተ፡ አሐቲ፡ ቤተ፡ ክርስቲያን፡ ዘሕንጽት፡ በስመ፡ ሰማዕት፡ መርቆሬዎስ፡ ወዝንቱሰ፡ ብእሲ፡ ሶበ፡ ኮነ፡ 
+                                        መዋዕሊ<cb n="b"/>ሁ፡ <hi rend="rubric">፪</hi>ዓመተ፡ ዖራ፡ <hi rend="rubric">፪</hi>ሆን፡
+                                        አዕይንቲሁ፡ ወነበረ፡ ዓመየ፡ ፍጹመ፡ እንዘ፡ 
+                                        <add place="above">ኢ</add>ይሬኢ<hi rend="rubric">፨</hi> ወሶበ፡ ኃጥአ፡ ብርሃነ፡ ለአዕይንቲሁ፡
+                                        ወጠነ፡ ከመ፡ ይግበር፡ ምህላ፡ እንዘ፡ ኢየአምሮ፡ <hi rend="rubric">፩</hi>፡ ወሀለወት፡ ውስተ፡ አሐቲ፡ ቤተ፡
+                                        ክርስቲያን፡ ሥዕለ፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> ማርያም፡ 
+                                    </incipit>
+                                    
+                                    <explicit xml:lang="gez">
+                                       <locus from="54ra" to="54rb"/>
+                                        ወአሠረ፡ ሀሊብኒ፡ ውስቴቶን፡ ወአንበሩ፡ እደዊሆሙ፡ ላዕለ፡ አዕይንቲሁ፡ ወተአምኁ<hi rend="rubric">፨</hi>
+                                        ወሖሩ፡ ኀበ፡ ሥዕላ፡ ለእግዝእትነ፡
+                                        <cb n="b"/>
+                                        ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡
+                                        አምላክ። ወሰገዱ<hi rend="rubric">፨</hi> ላቲ፡ <hi rend="rubric">፩፩</hi> አምጣነ፡ ክሂሎቶሙ፡
+                                        ወቀደሰ፡ ውእቱ፡ ቀሲስ፡ ዮሐንስ<hi rend="rubric">፤</hi> በይእቲ፡ ዕለት፡ ወመጠዎሙ፡ እምሥጢሩ፡ ቅዱስ፡ ለመድኃኔ፡
+                                        ዓለም፡ ለመነኮሳት<hi rend="rubric">፨</hi> ወነበረ፡ እምድኅረ፡ዝ፡ <hi rend="rubric">፳</hi> ዓመተ፡
```

Comment:
I think እምድኅረ፡ዝ፡ should still be typed እምድኅረ፡ ዝ፡ although they are very close (፡ was possibly added later in the middle).